### PR TITLE
[HAPI 2.0] Fix some broken validations

### PR DESCRIPTION
**Pull request by ashie**
And add some error messages.

### CODE REVIEW

File: server/common/HatoholArmPluginInterfaceHAPI2.cc [hunk 1]
```diff
@@ -46,81 +46,39 @@ std::list<HAPI2ProcedureDef> defaultValidProcedureList = {
...
-  : public AMQPMessageHandler
-{
-public:
-	AMQPHAPI2MessageHandler(HatoholArmPluginInterfaceHAPI2 &hapi2)
-	: m_hapi2(hapi2)
-	{
-	}
-
-	enum class JsonRpcObjectType {
+struct JsonRpcObject {
```
Comment (kz0817) [inline server/common/HatoholArmPluginInterfaceHAPI2.cc]: In other code, we use capital letters for abbreviated words such as JSON.
Here, we should use JSON_RPCObject or JSON_RPC_Object ?
But I feel a little uncomfortable...
JsonRpcObject looks simpler than JSON_RPCObject...

Comment (ashie) [inline server/common/HatoholArmPluginInterfaceHAPI2.cc]: > In other code, we use capital letters for abbreviated words such as JSON.

Yes, using "JSON" is our policy.
But I held a the same impressions as you:

> Here, we should use JSON_RPCObject or JSON_RPC_Object ?
> But I feel a little uncomfortable...
> JsonRpcObject looks simpler than JSON_RPCObject...

So, I used "JsonRpcObject".

But now I think we shouldn't change our policy.
I'll replace it with JSONRPCObject.

**Comment by ashie** [comment]
> I'll replace it with JSONRPCObject.

I did it.
Please continue your review.

**Comment by ashie** [comment]
Thanks!

**Pull request merged**